docs/usage/config-overview.md

```diff
@@ -1,12 +1,18 @@
 # Renovate configuration overview
 
-When Renovate runs on a repository, the final config used is derived from the:
-
-- Default config
-- Global config
-- Inherited config
-- Repository config
-- Resolved presets referenced in config
+When Renovate runs on a repository, the final config used is derived from the following config locations. The below is the order of precedence:
```

Comment:
```suggestion
Each time Renovate runs on a repository it reads the configuration files listed below and creates a final config.
This final config describes what Renovate will do during its run.

The final config is internal to Renovate, and is _not_ saved or cached for a later run.
But you can always find the final config in Renovate's logs.

Renovate reads the configuration files in this order (from from top to bottom):
```

I think we should explain that:

- the final config is _internal_ to Renovate
- and that Renovate does _not_ save/cache the final config for later (I think?)
- Renovate puts the final config in the log output 

Plus I'm rewriting bits we already had. 🙂